src/common/AutoComplete.scss

```diff
@@ -0,0 +1,51 @@
+.autocomplete-container {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  max-width: 400px;
+
+  .autocomplete-suggestion {
+    opacity: 0.5;
+    font-size: 1.05em;
+    cursor: pointer;
+    white-space: nowrap;
+    pointer-events: auto;
+    background: none;
+    border: none;
+    padding-top: 3px;
+    padding-left: 0px;
+    margin-left: -3px;
+  }
+
+  .autocomplete-input-wrapper {
+    position: relative;
+    display: inline-flex;
+    align-items: center;
+    width: 100%;
+    padding: 0;
+    background: none;
```

Comment:
check **simple** and **place-middle** in short.scss

Reply:
cant use place-middle here as we want to align on vertically not placing it in center